opentelemetry-zio-logging/src/main/scala/zio/telemetry/opentelemetry/zio/logging/TelemetryLogFormats.scala

```diff
@@ -0,0 +1,39 @@
+package zio.telemetry.opentelemetry.zio.logging
+
+import io.opentelemetry.api.trace.{Span, SpanContext}
+import io.opentelemetry.context.Context
+import zio.FiberRefs
+import zio.logging.LogFormat
+import zio.telemetry.opentelemetry.context.ContextStorage
+
+object TelemetryLogFormats {
```

Comment:
We should leverage the ZIO Module Pattern 2.0 in this case as well as we do for other parts (e.g. [Tracing](https://github.com/zio/zio-telemetry/blob/series/2.x/opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala#L519). Basically, it means we need to pass `ctxStorage` as a constructor's argument of `TelemetryLogFormats` class and add a `TelemetryLogFormats.live` ZLayer dependant on the `ContextStorage`.

Reply:
Fixed